src/Microsoft.Health.Dicom.Functions.Client/DicomAzureFunctionsClient.cs

```diff
@@ -146,17 +148,77 @@ public async Task<OperationReference> StartExportAsync(Guid operationId, ExportS
     }
 
     /// <inheritdoc/>
-    public Task<Guid> StartBlobCopyAsync(CancellationToken cancellationToken = default)
+    public async Task StartBlobCopyAsync(CancellationToken cancellationToken = default)
```

Comment:
I think we should simplify this a bit. Let's not have the ` DicomAzureFunctionsClient` know about a special known copy operation. Instead, it should have a `StartBlobCopyAsync` method like all the others that takes an ID (and perhaps doesn't output anything).

Then your background service can know about the special ID and include retry logic in case of failures. It just relies upon some `IDicomOperationsClient` instead.

Reply:
Done